Jarvis-Platform/jarvis-studio
# [Feature]: Linter configuration & fixes

# Whats 🆕 ?
- Created all necessary configuration files
- Fixed all errors & warnings

# How to test?
- `npm run serve` and see no error in console & in terminal.
- Try functionalities but none of them has to be impacted because fixes have been made on "code style"
